src/parser.js

```diff
@@ -1354,6 +1361,13 @@ export default class Parser {
     _.forEach(params, (parameter, index) => {
       this.withPath('parameters', index, () => {
         let member;
+        const format = parameter.collectionFormat || 'csv';
+
+        // Adding a warning if format is not supported
+        if (['multi', 'csv'].indexOf(format) === -1) {
+          this.createAnnotation(annotations.DATA_LOST, this.path,
+            `Parameters of collection format '${format}' are not yet supported`);
```

Comment:
I'd write this message as less of a promise "not yet supported". This message would imply we will support it, which we may never do.

```diff
- `Parameters of collection format '${format}' are not yet supported`);
+ `Parameters of collection format '${format}' are not supported`);
```